README.md

```diff
@@ -84,15 +84,16 @@ First, you'll want to have Node.js installed. You can do this by going to [nodej
 
 6. Run `npm run contributors:generate` in your terminal to generate the guestbook on the README.
 
-7. If you used the CLI tool method in step 6, to add yourself to the guestbook, **skip this step;** otherwise:
-   Add and commit your changes by running this command:
```

Comment:
We don't want to remove line 87-88 as these are our newest docs update.

What you need to do:
- Copy and paste these lines and put it back to where the were.

Reply:
As I synced fork and pull in my local repo, it is like that now in the readme file